### PR TITLE
Allow _resolvePlural to search _fallbackTranslation values when trying to resolve a plural key

### PR DESCRIPTION
**Pull request by shrdnar**
Not sure of the logic/reasoning behind line 181, where the call to `_resolve` is forcing `fallback` param to always be false.
That means if you **are** using fallback translations (`_fallbackTranslations` is not null and has values) `_resolvePlural` should respect that and attempt to resolve any plural keys that exist in the fallback translations map.

### CODE REVIEW

File: lib/src/localization.dart [hunk 1]
```diff
@@ -178,7 +178,7 @@ class Localization {
     if (subKey == 'other') return _resolve('$key.other');
 
     final tag = '$key.$subKey';
-    var resource = _resolve(tag, logging: false, fallback: false);
+    var resource = _resolve(tag, logging: false, fallback: _fallbackTranslations != null);
```
Comment (bw-flagship) [inline lib/src/localization.dart:181]: It would be greaf if you could add a test for this

File: pubspec.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ dependencies:
   flutter:
     sdk: flutter
   shared_preferences: '>=2.0.0 <3.0.0'
-  intl: '>=0.17.0-0 <=0.18.1'
+  intl: '>=0.18.0-0 <=0.19.0'
```
Comment (bw-flagship) [inline pubspec.yaml]: Please revert -> if we want to do that, then in a seperate PR

**Review by bw-flagship — CHANGES_REQUESTED** [review]
This change seems to make sense, thanks for the PR! However, a test would be good to avoid breaking it in the future

File: lib/src/localization.dart (hunk 1 above)
Comment (shrdnar) [reply lib/src/localization.dart:181]: Updated test case

**Review by bw-flagship — APPROVED** [review]

**Pull request merged**